src/social_auth/backends/github.py

```diff
@@ -30,19 +30,27 @@
 from social_auth.utils import dsa_urlopen
 from social_auth.backends import BaseOAuth2, OAuthBackend
 from social_auth.exceptions import AuthFailed
+import  os
+
+GITHUB_BASE_DOMAIN='github.com'
```

Comment:
We should definitely not have this configured via environment variables. We can move them into normal settings.